tools/simg2dontcare.py

```diff
@@ -1,76 +0,0 @@
-#!/usr/bin/env python3
-import argparse
```

Comment:
we need to keep skip-chunks around for flash.comma.ai. this PR should just be for the openpilot OTA image

Reply:
got it, wasn’t sure we wanted to keep it or not.

- I assumed that if we need to push an AGNOS 10.2 update before the 11 release, we could just fork from 10.1 and push any patches on a 10.2 hotfix branch. used this “workflow” before in other projects when an intermediary update was needed while working at the next major release (that wasn’t ready for release).

- 2nd point about comma flash, we can just fake the “alt” image to be the same image as the main (raw) one - so comma flash doesn’t crash looking for the “alt” image.

these two points could make sense so that we don’t spam GitLab LFS with two 800 MB files per push.

but I agree with you that having both images is a much cleaner (even if getting close to the 10 GB limit on GitLab LFS)